CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+## How to contribute to Psych
+
+#### **Did you find a bug?**
+
+* **Do not open up a GitHub issue if the bug is a security vulnerability
+  in Psych**, instead refer to our [security policy](https://www.ruby-lang.org/en/security/).
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/ruby/ruby/issues) or [Psych Issues](https://github.com/ruby/psych/issues).
```

Comment:
[ruby/ruby](https://github.com/ruby/ruby) does not use issues on GitHub, and use [redmine](https://bugs.ruby-lang.org/) as main issue tracker.

Reply:
Oh I was actually uncertain wether we wanted reports directly to Psych or everything to ruby :)

Thanks! I'll fix this immediately